scout/server/blueprints/variant/templates/variant/cancer-variant.html

```diff
@@ -0,0 +1,659 @@
+{% extends "layout_bs4.html" %}
+{% from "utils.html" import comments_panel, activity_panel, pedigree_panel %}
+{% from "variants/utils.html" import compounds_table, modal_causative %}
+{% from "variant/utils.html" import rankscore_panel, overlapping_panel, genes_panel, transcripts_panel, proteins_panel,  pin_button, causative_button, igv_track_selection %}
+{% from "variant/tx_overview.html" import disease_associated, transcripts_overview %}
+{% from "variant/variant_details.html" import frequencies, gtcall_panel, observations_panel, conservations %}
+{% from "variant/buttons.html" import igv_button, variant_tag_button, variant_tier_button, dismiss_variant_button, mosaic_variant_button, splice_junctions_button %}
+
+{% block title %}
+  {{ super() }} - {{ institute.display_name }} - {{ case.display_name }} - {{ variant.display_name }}
+{% endblock %}
+
+{% block css %}
+{{ super() }}
+<link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap-select@1.13.9/dist/css/bootstrap-select.min.css">
+{% endblock %}
+
+{% block top_nav %}
+  {{ super() }}
+  <li class="nav-item">
+    <a class="nav-link text-nowrap" href="{{ url_for('overview.cases', institute_id=institute._id) }}">
+      {{ institute.display_name }}
+    </a>
+  </li>
+  <li class="nav-item">
+    <a class="nav-link text-nowrap" href="{{ url_for('cases.case', institute_id=institute._id, case_name=case.display_name) }}">
+      {{ case.display_name }}
+    </a>
+  </li>
+  <li class="nav-item">
+    {% if cancer %}
+      <a class="nav-link text-nowrap" href="{{ url_for('variants.cancer_variants', institute_id=institute._id, case_name=case.display_name, variant_type=variant.variant_type, gene_panels=case.panels|selectattr('is_default')|map(attribute='panel_name')|list) }}">
+        {{ variant.variant_type|capitalize }} cancer variants
+      </a>
+    {% elif str %}
```

Comment:
> I also like how the Classification and Summary panel acquire a specific purpose and I think it should be the same also in non-cancer variant! 👍

I can "port" the summary view to the "normal" variant page as well although that would increase the scope of this rather big PR. Would you be ok with it being included in this PR as well or should it be a stand alone PR?

> As soon as I've started reviewing the new template I've noticed that almost everything it contains is a copy of the variant.html one. Since most of the content is shared (for now) it makes total sense to me to have those macros into another file, to be imported by both variant.html and cancer-variant.html.

True. I kept them separate as they might diverge further in the future but I am open minimize the duplication of code.

Reply:
> I can "port" the summary view to the "normal" variant page as well although that would increase the scope of this rather big PR. Would you be ok with it being included in this PR as well or should it be a stand alone PR?

Summary could be a macro used by both types of variants, no need to have it duplicated also on variant.html